generator/src/main/java/com/emc/mongoose/generator/BasicGenerator.java

```diff
@@ -184,11 +194,14 @@ public BasicGenerator(
 		
```

Comment:
It is great, when a class constructor is placed right after fields, but not in the middle of a listing.